modules/base/errors/module.py

```diff
@@ -49,308 +51,662 @@ async def on_command_error(
+                True,
+            )
+        # Just in case we missed something
+        else:
+            return (
+                _(ctx, "Nextcord library error"),
+                _(ctx, "HTTP Exception"),
                 False,
             )
-        if type(error) == commands.BadBoolArgument:
+
+    @staticmethod
+    async def handle_ExtensionError(ctx, error) -> Tuple[str, str, bool]:
+        """Handles exceptions raised by pumpkin-py extensions
+
+        Args:
+            ctx: The invocation context.
+            error: Detected exception.
+
+        Returns:
+            Tuple[str, str, bool]: Translated error name, Translated description, Whether to ignore traceback in the log
+        """
+        # return friendly name, e.g. strip "modules.{module}.module"
+        name = error.name[8:-7]
+        # send helpful message if the requested module does not follow naming rules
+        if re.fullmatch(r"([a-z_]+)\.([a-z_]+)", name) is None:
+            description = _(
+                ctx,
+                "The extension **{extension}** could not be found. It should be in `repository.module` format",
+            ).format(
+                extension=name,
+            )
+        # An exception raised when an extension has already been loaded.
+        elif isinstance(error, commands.ExtensionAlreadyLoaded):
+            description = _(ctx, "Extension **{extension}** is already loaded").format(
+                extension=name,
+            )
+        # An exception raised when an extension was not loaded.
+        elif isinstance(error, commands.ExtensionNotLoaded):
+            description = _(ctx, "The extension **{extension}** is not loaded").format(
+                extension=name,
+            )
+        # An exception raised when an extension does not have a setup entry point function.
+        elif isinstance(error, commands.NoEntryPointError):
+            description = _(
+                ctx, "Extension **{extension}** doesn't have a setup entry function"
+            ).format(
+                extension=name,
+            )
+        # An exception raised when an extension failed to load during execution of the module or setup entry point.
+        elif isinstance(error, commands.ExtensionFailed):
+            description = _(ctx, "**{extension}** failed").format(extension=name)
+        # An exception raised when an extension is not found.
+        elif isinstance(error, commands.ExtensionNotFound):
+            description = _(
+                ctx, "The extension **{extension}** could not be found"
+            ).format(
+                extension=name,
+            )
+        # Just in case we missed something
+        else:
             return (
-                _(ctx, "Bad Boolean Argument"),
-                _(ctx, "Argument `{arg}` is not binary").format(arg=error.argument),
+                _(ctx, "Nextcord library error"),
+                _(ctx, "Extension error"),
```

Comment:
I'm setting the `ignore_errors` attribute to False here so the full traceback should get senf as well. I don't think anything more is necessary as the developer would end up looking at the traceback anyway. 

These `else` handlings like this one are there just in case a new exception class is added to the library anyway since every current exception is being caught.
So theoretically we should never get there unless something changes.